tests/by-util/test_expr.rs

```diff
@@ -123,6 +123,11 @@ fn test_or() {
         .args(&["-14", "|", "1"])
         .succeeds()
         .stdout_only("-14\n");
+
```

Comment:
could you please add more tests to test this behavior ? :)
thanks

Reply:
Add two more tests, and they can be passed on my local machine :)